sklearn/preprocessing/data.py

```diff
@@ -1451,7 +1451,7 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
     >>> enc = OneHotEncoder()
     >>> enc.fit([[0, 0, 3], [1, 1, 0], [0, 2, 1], \
 [1, 0, 2]])  # doctest: +ELLIPSIS
-    OneHotEncoder(categorical_features='all', dtype=<... 'float'>,
+    OneHotEncoder(categorical_features='all', dtype=<... 'numpy.float64'>,
```

Comment:
As far as my understanding goes... `numpy.float` is alias of python's default `float` while `numpy.float64` is a scalar type defined by numpy...
(This [answer](http://stackoverflow.com/a/16964006/3109769) seems to explain it... (especially the comment below it!))